composer.json

```diff
@@ -33,12 +33,14 @@
   , "require": {
         "php": "^7.2"
       , "ratchet/rfc6455": "^0.3"
-      , "react/socket": "^1.0 || ^0.8 || ^0.7 || ^0.6 || ^0.5"
```

Comment:
I'd like to keep these in here for v0.5. v0.6 will drop support for old versions.